includes/class-wasa-kredit-invoice-checkout-payment-gateway.php

```diff
@@ -132,7 +132,7 @@ public function init_form_fields() {
 				'title'   => __( 'Enable Order Management', 'wasa-kredit-checkout' ),
 				'type'    => 'checkbox',
 				'label'   => __( 'Enable Wasa Kredit order capture on WooCommerce order completion.', 'wasa-kredit-checkout' ),
-				'default' => 'yes',
+				'default' => 'no',
```

Comment:
The setting should default to yes.

Reply:
Solved in https://github.com/krokedil/woocommerce-plugin/pull/2/commits/5056f17d8117bb06a8055f78393036cfcd12d5fb